src/app/Library/CrudPanel/CrudButton.php

```diff
@@ -159,6 +162,19 @@ public function view($value)
         return $this->save();
     }
 
+    /**
+     * Sets the parameters that will be sent to the view.
+     *
+     * @param  array  $value  Path to view file.
+     * @return CrudButton
+     */
+    public function parameters($value)
```

Comment:
I still think `configuration` or `meta` makes more sense than `parameters`.

It's not url parameters, neither function parameter/arguments

Reply:
Ok you convinced me. `meta` does seem better, let's go with that. Changed!